packages/keyv-redis/src/index.ts

```diff
@@ -0,0 +1,89 @@
+import EventEmitter from 'events';
+import { promisify as pify } from 'util';
+import redis from 'redis';
+import { KeyvStore } from 'keyv';
+
+interface Client {
+	del(...keys: string[]): Promise<number>;
+	get(key: string): Promise<string>;
+	sadd(namespace: string, key: string): Promise<number>;
+	srem(namespace: string, key: string): Promise<unknown>;
+	smembers(namespace: string): Promise<string[]>;
+
+	set(key: string, value: string): Promise<unknown>;
+	set(key: string, value: string, mode: string, ttl: number): Promise<unknown>;
+}
+
+export interface KeyvRedisOptions extends redis.ClientOpts {
```

Comment:
this is fairly self-explanatory, but it adds the interface of redis's community supported types to this interface. lets consumers get auto-complete for redis options without needing to list them all and maintain them here.